File: AccountsService/Models/message.go
package models

import (
	"time"

	"go.mongodb.org/mongo-driver/bson/primitive"
)

type Message struct {
	ID         primitive.ObjectID `bson:"_id,omitempty" json:"id,omitempty"`
	ChatId     primitive.ObjectID `bson:"chat_id" json:"chat_id"`
	SenderId   primitive.ObjectID `bson:"sender_id" json:"sender_id"`
	Text       string             `bson:"text" json:"text"`
	Created    time.Time          `bson:"created" json:"created"`
	IsModified bool               `bson:"is_modified" json:"is_modified"`
}

File: Core/RiffCore/RiffCore/Models/Message.cs
using System.Text.Json.Serialization;

namespace RiffCore.Models;

public class Message
{
    [JsonPropertyName("id")]
    public string Id { get; set; }
    [JsonPropertyName("chat_id")]
    public string ChatId { get; set; }
    [JsonPropertyName("sender_id")]
    public string SenderId { get; set; }
    [JsonPropertyName("text")]
    public string Text { get; set; }
    [JsonPropertyName("created")]
    public DateTime Created { get; set; }
    [JsonPropertyName("is_modified")]
    public bool IsModified { get; set; }
    [JsonPropertyName("correlation_id")]
    public string CorrelationId { get; set; }
    
    
}